languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/disambiguation.xml

```diff
@@ -3167,6 +3167,264 @@ RB => Règles Brutes
+  <rule id="TOUT" name="tout">
+    <pattern>
+      <marker>
+        <token>tout</token>
+      </marker>
+      <token regexp="yes">au|du</token>
+      <token regexp="yes">moins|plus</token>
+    </pattern>
+    <disambig action="filter" postag="A"/>
+  </rule>
+  <rule id="COURT_LE_LONG" name="court le long">
+    <pattern>
+      <marker>
+        <token postag="V.*" postag_regexp="yes" inflected="yes" skip="1">court</token>
+      </marker>
+      <token>long</token>
+      <token>de</token>
+    </pattern>
+    <disambig action="filter" postag="V"/>
+  </rule>
+  <rule id="COURT_A_SA_PERTE" name="courir à sa perte">
+    <pattern>
+      <marker>
+        <token regexp="yes">court</token>
+      </marker>
+      <token skip="1">à</token>
+      <token>perte</token>
+    </pattern>
+    <disambig action="filter" postag="V"/>
+  </rule>
+  <rule id="TIRER_PARTIE" name="tirer partie">
+    <pattern>
+      <marker>
+        <token postag="V.*" postag_regexp="yes" inflected="yes">tirer</token>
+      </marker>
+      <token>partie</token>
+    </pattern>
+    <disambig action="filterall"/>
+  </rule>
+  <rule id="RUMEUR_COURT" name="rumeur court">
+    <pattern>
+      <token>rumeur</token>
+      <marker>
+        <token postag="V.*" postag_regexp="yes" inflected="yes">court</token>
+      </marker>
+    </pattern>
+    <disambig action="filter" postag="V"/>
+  </rule>
+  <rule id="COURT_LE_RISQUE" name="court le risque">
+    <pattern>
+      <marker>
+        <token postag="V.*" postag_regexp="yes" inflected="yes" skip="1">court</token>
+      </marker>
+      <token>le</token>
+      <token>risque</token>
+    </pattern>
+    <disambig action="filter" postag="V"/>
+  </rule>
+  <rule id="SE_SAVOIR" name="se savoir">
+    <pattern>
+      <token postag="R pers obj.*" postag_regexp="yes"/>
+      <marker>
+        <token>savoir</token>
+      </marker>
+    </pattern>
+    <disambig action="filter" postag="V"/>
+  </rule>
+  <rule id="RENDRE_VISITE" name="rendre visite">
+    <pattern>
+      <token postag="V.*" postag_regexp="yes" inflected="yes">rendre</token>
+      <token>visite</token>
+    </pattern>
+    <disambig action="filterall"/>
+  </rule>
+  <rule id="METRES_COURSE" name="mètres dans une course">
+    <pattern>
+      <token postag="Y"/>
+      <token>mètres</token>
+      <token regexp="yes">féminin|masculin</token>
+    </pattern>
+    <disambig action="filterall"/>
+  </rule>
+  <rule id="COTE_NOM" name="côté nom">
+    <pattern>
+      <token postag="[JD].*" postag_regexp="yes"/>
+      <marker>
+        <token regexp="yes">c[oô]t[ée]</token>
+      </marker>
+      <token postag="N.*" postag_regexp="yes"/>
+    </pattern>
+    <disambig action="filter" postag="N.*"/>
+  </rule>
+  <rule id="GAGNE" name="gagne">
+    <pattern>
+      <token postag="N.*" postag_regexp="yes"/>
+      <marker>
+        <token regexp="yes">gagne</token>
+      </marker>
+    </pattern>
+    <disambig action="filter" postag="V.*"/>
+  </rule>
```

Comment:
In cases like this, it is better to make sure that the desired POS tag is still there: 
```
<marker>
  <token postag="V.*" postag_regexp="yes">gagne</token>
</marker>
```
`regexp=yes"` is not needed.